doc/customMetrics.md

```diff
@@ -0,0 +1,24 @@
+# Custom metrics
+
+Our application uses App Insights, an Azure service, to monitor some user actions and webapp performance. This documentation lists the custom metrics and what user actions trigger each of them.
+For further information on how to consult these custom metrics, see: [How to view logs and metrics in Application Insights](https://azure.github.io/Industrial-IoT/tutorials/tut-applicationinsights.html#view-metrics-in-application-insights).
+
+## Upload
+
+This event is triggered each time a user uploads a file, by clicking on 'upload' button, within a Scenario Parameters tab.
+
+## Download
+
+This event is triggered each time a user downloads a file, by clicking on 'upload' button, within a Scenario Parameters tab.
+
+## Create a new scenario
+
+This event is triggered each time a user creates a new scenario, i.e. when clicking on the 'create' button from the scenario creation dialog.
+
+## Launch a scenario
+
+This event is triggered each time a user launches a scenario, either by clicking 'launch' or 'update and launch' buttons.
+
+## Duration of a scenario run
+
+This event is triggered each time a user launches a scenario, either by clicking 'launch' or 'update and launch' button. The duration is computed from the start date and the end date, when the run is successful.
```

Comment:
The duration is measured between the moment the user has clicked on the button, and the moment the application has detected the related scenario status to be either "Successful" or "Error". 

Note: The measured run duration can be higher than the real duration. After scenario launch, the application will check the scenario's status value regularly, at a time interval set by variable SCENARIO_STATUS_POLLING_DELAY ([see App Behavior configuration](https://github.com/Cosmo-Tech/azure-sample-webapp/blob/main/doc/appConfiguration.md#edit-configuration-files)). This is why the run duration measurement error is inferior or equal to SCENARIO_STATUS_POLLING_DELAY.